space2stats_api/src/tests/test_api.py

```diff
@@ -51,13 +49,22 @@ def client(monkeypatch, database, test_bucket):
         yield app
 
 
+@pytest.fixture
+def mock_get_summaries(mocker):
+    return mocker.patch("space2stats.main._get_summaries")
+
+
+@pytest.fixture
+def mock_get_available_fields(mocker):
+    return mocker.patch("space2stats.app.get_available_fields")
```

Comment:
This won't close what I suggested. This still use `mock` responses. I meant that we should ingest data in a database like we do in https://github.com/developmentseed/tipg/blob/b617fa9a45bce574ef247863d93208c8ea6f0e22/tests/conftest.py#L22-L101